canonical/ubuntu.com
# /desktop/flavours refresh

## Done

- Applied refresh as per [doc](https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit) and [design](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?node-id=498%3A14707&mode=dev)

## QA

- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/flavours
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7746
